examples/src/main/r/RSparkSQLExample.R

```diff
@@ -118,6 +118,10 @@ df <- read.df("examples/src/main/resources/people.csv", "csv", sep=";", inferSch
 namesAndAges <- select(df, "name", "age")
 # $example off:manual_load_options_csv$
 
+# $example on:manual_save_options_orc$
+df <- read.df("examples/src/main/resources/users.orc", "orc")
+write.orc(df, "users_with_options.orc", orc.bloom.filter.columns="favorite_color", orc.dictionary.key.threshold=1.0, orc.column.encoding.direct="name")
+# $example off:manual_save_options_orc$
```

Comment:
@felixcheung . Could you review this PR?

Reply:
we should put space after param
(gosh same for csv example above)

`orc.bloom.filter.columns = "favorite_color", orc.dictionary.key.threshold = 1.0, orc.column.encoding.direct = "name")`

Reply:
Thank you!